torchci/rockset/commons/__sql/individual_test_times_per_oncall_per_workflow.sql

```diff
@@ -0,0 +1,95 @@
+WITH
+    filtered_table AS (
+        SELECT
+            oncalls,
+            workflow_name,
+            CAST(workflow_id as STRING) as workflow_id,
+            test_class,
+            test_file,
+        FROM
+            metrics.aggregated_test_metrics
+        WHERE
+            workflow_run_attempt = 1
+            AND workflow_id IS NOT NULL
+            AND workflow_name = :workflow_name
+            AND DATE_TRUNC('DAY', PARSE_DATETIME_ISO8601(date)) = DATE_TRUNC('DAY', CAST(PARSE_DATETIME_ISO8601(:queryDate) as date))
+            AND test_class LIKE :classname
+        GROUP BY
```

Comment:
I'm a bit curious about the effect of this GROUP BY as it groups everything.  Would a `SELECT DISTINCT` suffice?  I wonder which ways are faster, maybe they are the same